### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andriykooo**
[DEMO LINK](https://Andriykooo.github.io/react_tabs/)

### CODE REVIEW

File: src/components/Tabs.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <ul className="tabs__list">
+      {tabs.map(tab => (
+        <li key={tab.id} className="tabs__item">
+          <button
+            type='button'
+            className={
+              `tabs__btn ${tab.id !== selectedTabId
+                ? 'tabs__btn--active'
+                : ''}`
+            }
```
Comment (illisogori) [inline src/components/Tabs.jsx:21]:
```suggestion
            className={
              `tabs__btn ${tab.id !== selectedTabId
                ? 'tabs__btn--active'
                : ''}`
            }
```
Можно использовать Classnames.

**Review by illisogori — APPROVED** [review]
Отлично.

File: src/components/Tabs.jsx (hunk 1 above)
Comment (VinnieJ-2k20) [reply src/components/Tabs.jsx:21]: Не можно, а нужно 😄

File: src/components/Tabs.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    &--active {
+      background-color: transparent;
+    }
+  }
+
+  &__content {
+    color: #121213;
+  }
+}
```
Comment (VinnieJ-2k20) [inline src/components/Tabs.scss:41]: Don't forget about extra lines at the end.

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍 

Added some comments.

**Pull request closed without merging**